src/app/information/information.component.ts

```diff
@@ -11,51 +13,38 @@ import { changeInfoPropertiesAction } from '@app/episode/info/store/info.actions
   styleUrls: ['./information.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class InformationComponent implements OnInit {
-  episodeInfo$ = this.store.select(selectEpisodeInfo).pipe(
-    tap((value) => console.log('Episode Info', value)),
-    startWith(emptyEpisodeInfo()),
-    tap((value) => console.log('Episode Info', value)),
-  );
+export class InformationComponent implements OnInit, OnDestroy {
+  /**
+   * Auto-save fields might fail, this flag will determine if there is something unsaved
+   */
+  isPendingUpdates = false;
+  /**
+   * UI Model for binding episode info
+   */
+  episodeInfo: EpisodeInfo;
+  /**
+   * Episode Info subscription from store
+   */
+  private episodeInfo$: Subscription;
 
   constructor(private store: Store) {}
 
-  ngOnInit(): void {}
+  ngOnDestroy(): void {
+    // Avoid memory leaks, must unsubscribe
+    this.episodeInfo$.unsubscribe();
+  }
+
+  ngOnInit(): void {
+    this.episodeInfo$ = this.store
+      .select(selectEpisodeInfo)
+      .pipe(take(1)) // No need to keep updating
```

Comment:
Angular destroys a component once you deactivate it (eg.: navigate away), so `ngOnInit` is going to be called only once, and every time the user opens this page.

Reply:
I'd love to discuss this in our next phone call. I see the concept but would like to better understand the specifics.